From 22731728d6ccc2c41c9cea327a2e51eade869144 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
To: Andrew Morton <akpm@linux-foundation.org>
Cc: damon@lists.linux.dev
Cc: linux-mm@kvack.org
Cc: linux-kernel@vger.kernel.org
Date: Thu, 22 Aug 2024 13:58:09 -0700
Subject: [PATCH 3/4] mm/damon/core: remove per-scheme region priority
 histogram buffer

Nobody is reading from or writing to the per-scheme region priorities
histogram buffer.  It is only wasting memory.  Remove it.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 include/linux/damon.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/linux/damon.h b/include/linux/damon.h
index 6342d8f9b0fd..0ac8b862bdcb 100644
--- a/include/linux/damon.h
+++ b/include/linux/damon.h
@@ -242,7 +242,6 @@ struct damos_quota {
 	unsigned long charge_addr_from;
 
 	/* For prioritization */
-	unsigned long histogram[DAMOS_MAX_SCORE + 1];
 	unsigned int min_score;
 
 	/* For feedback loop */
-- 
2.39.2

